### ISSUE TITLE
Error(s) in loading state_dict for SequenceTagger: Unexpected key(s) in state_dict: "lstm_init_h", "lstm_init_c".

### ISSUE THREAD

**Issue by ambuje** [flairNLP/flair#3009]
from flair.models import SequenceTagger
tagger : SequenceTagger = SequenceTagger(hidden_size=128,
                                       embeddings=embeddings,dropout=0.2,locked_dropout=0.2,
                                       tag_dictionary=tag_ditionary1,train_initial_hidden_state=True,
                                       tag_type=tag_type,rnn_layers=4,tag_format='BIO',
                                       use_crf=False)

If train_initial_hidden_state=True, then we receive the error:- 

Error(s) in loading state_dict for SequenceTagger: 
Unexpected key(s) in state_dict: "lstm_init_h", "lstm_init_c".

Task - NER

Can you please help me out?

Thank you

**Comment by helpmefindaname** [comment]
Hi @ambuje 
seems like this is currently not possible, I created a PR to fix it, can you try out the following: https://github.com/flairNLP/flair/pull/3010 and see if that works for you?
